spec/integration/mentioning_spec.rb

```diff
@@ -28,6 +28,7 @@ def users_connected?(user1, user2)
 
 describe "mentioning", type: :request do
   include MentioningSpecHelpers
+  include RequestsHelper
```

Comment:
Is this the only place where this is used? Why do we need a Helper then, when it is only used once?